src/vivarium_public_health/metrics/stratification.py

```diff
@@ -37,22 +37,22 @@ def register_stratifications(self, builder: Builder) -> None:
             is_vectorized=True,
             requires_columns=["current_time"],
         )
-        builder.results.register_stratification(
-            "event_year",
-            [str(year) for year in range(self.start_year, self.end_year + 1)],
-            self.map_year,
-            is_vectorized=True,
-            requires_columns=["event_time"],
-        )
+        # builder.results.register_stratification(
```

Comment:
Do we want to include [MIC-4232](https://jira.ihme.washington.edu/browse/MIC-4232) as a TODO?

Reply:
Done